lib/datadog.py

```diff
@@ -0,0 +1,113 @@
+import os
+import json
+import yaml
+import subprocess
+import buildpackutil
+
+
+def is_enabled():
+    return os.getenv('DD_API_KEY') is not None
+
+
+def _get_hostname():
+    domain = buildpackutil.get_vcap_data()['application_uris'][0].split('/')[0]
+    return domain + '-' + os.getenv('CF_INSTANCE_INDEX', '')
+
+
+def update_config(m2ee, app_name):
+    if not is_enabled():
+        return
+    tags = json.loads(os.getenv('DD_TAGS', '[]'))
+    m2ee.config._conf['m2ee']['javaopts'].extend([
+        '-Dcom.sun.management.jmxremote',
+        '-Dcom.sun.management.jmxremote.port=7845',
+        '-Dcom.sun.management.jmxremote.local.only=true',
+        '-Dcom.sun.management.jmxremote.authenticate=false',
+        '-Dcom.sun.management.jmxremote.ssl=false',
+        '-Djava.rmi.server.hostname=127.0.0.1',
+    ])
+    m2ee.config._conf['logging'].append({
+        'type': 'file',
+        'name': 'FileSubscriberDataDog',
+        'autosubscribe': 'INFO',
+        'filename': os.path.join(os.getcwd(), 'log', 'datadog.log'),
+        'max_size': 1048576,
```

Comment:
Shouldn't this also match #158 ?

Reply:
No, here the runtime will actually handle file rotation because it's a regular file (not a fifo).